esp8266-fastled-webserver/include/simplehacks/compile_date.h

```diff
@@ -0,0 +1,136 @@
+/**
```

Comment:
SIMPLE FILE REVIEW

Allows the date and time of compilation to be added as `constexpr` compatible string.  Will be used in `/product` JSON endpoint.